src/custom-element.js

```diff
@@ -40,6 +40,7 @@ import {
 import * as dom from './dom';
 import {setStyle, setStyles} from './style';
 import {LayoutDelayMeter} from './layout-delay-meter';
+import {ResourceState} from './service/resource';
```

Comment:
That by itself is not too bad. But why? Don't we only toggle fallback from the element itself? How can it be not built yet? Also, internal state in this class contains most of this data already: `isBuilt`, `isFirstLayoutCompleted_`, `layoutCount_` and `layoutWidth_`. So, in theory, this is not very necessary.